streaming/stream.conf

```diff
@@ -17,7 +17,7 @@
     # Where is the receiving netdata?
```

Comment:
As discussed in the demo, we need a few more things here. 
Check section "Certificate Verification" in https://curl.haxx.se/docs/sslcerts.html, it covers all the cases. 
We will have the exact same thing when netdata acts as a client (i.e. in backends now and later when a netdata agent tries to connect to https://netdata.cloud).

Reply:
We can accept just the skip verification to merge this, but please create a new issue so we can add more options later. Put here a reference to that issue and we can resolve this.